e2e/claims.service.e2e.ts

```diff
@@ -948,6 +963,36 @@ describe('Сlaim tests', () => {
         ).toBe(true);
       });
 
+      test('should be able to issue without request and publish onchain when signer is not subject (i.e. for asset)', async () => {
+        const claimType = `${roleForAsset}.${root}`;
+        const claim = await issueWithoutRequest(rootOwner, {
+          subjectDID: myAssetDid,
+          claimType,
+          registrationTypes,
+        });
+        expect(claim.onChainProof).toHaveLength(132);
+        await signerService.connect(rootOwner, ProviderType.PrivateKey);
+        const mockedClaim = {
+          claimType,
+          isApproved: true,
+          onChainProof: claim.onChainProof,
+          claimTypeVersion: version,
+          acceptedBy: claim.acceptedBy,
+          subject: myAssetDid,
+        };
+        mockGetClaimsBySubject
+          .mockReset()
+          .mockImplementationOnce(() => [mockedClaim]);
+
+        await claimsService.publishPublicClaim({
```

Comment:
@whitneypurdum from what I can tell, in the test case, the asset DID is a random end account and not an asset smart contract. This is the code that gives me this impression:
```
const myAsset = Wallet.createRandom().connect(provider);
const myAssetDid = `did:${Methods.Erc1056}:${Chain.VOLTA}:${myAsset.address}`;
```

Am I reading this right? I think that the test needs to actually use the asset contracts so that the contracts are there to represent the ownership relationship